gui/tasks/electron.js

```diff
@@ -0,0 +1,23 @@
+const { spawn } = require('child_process');
+const electron = require('electron');
+
+let subprocess;
+
+function startElectron(done) {
+  subprocess = spawn(electron, ['.'], {
+    env: { ...process.env, NODE_ENV: 'development' },
+    stdio: 'inherit',
+  });
+  done();
+}
+
+function stopElectron() {
```

Comment:
FYI Gulp tasks automatically handle streams, subprocesses and promises when they are returned back form tasks. i.e it waits for async task to complete. The other way to handle this is to accept a callback argument and call it later.